test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTrimTest.java

```diff
@@ -78,11 +82,11 @@ public void ensureMCWUsesRealTail() throws Exception {
         }
 
         MultiCheckpointWriter mcw = new MultiCheckpointWriter();
-        mcw.addMap((SMRMap) map);
+        mcw.addMap(map);
         Token trimAddress = mcw.appendCheckpoints(getRuntime(), "author");
         Token staleTrimAddress = new Token(trimAddress.getEpoch() - 1, trimAddress.getSequence());
-        assertThatThrownBy(() -> getRuntime().getAddressSpaceView().prefixTrim(staleTrimAddress))
-                .isInstanceOf(WrongEpochException.class);
+
+        getRuntime().getAddressSpaceView().prefixTrim(staleTrimAddress);
```

Comment:
CAn you also check that LogUnitServer TRIM_MARK_REQUEST gives you correct/expected trim mark address

Reply:
Done.